framework/src/task_manager/Kconfig

```diff
@@ -14,6 +14,12 @@ config TASK_MANAGER
 
 if TASK_MANAGER
 
+config TASK_MANAGER_DRVPATH
```

Comment:
Please determin whether user can change this config or not and move this to appropriate location.